.github/ISSUE_TEMPLATE.md

```diff
@@ -1,5 +1,8 @@
 <!--
 
+For Windows PowerShell 5.1 issues, suggestions, or feature requests please use the following link instead:
+https://windowsserver.uservoice.com/forums/301869-powershell
```

Comment:
Good idea!  Can you format line 4 to be similar to line 10 below?

```markdown
- Windows PowerShell [UserVoice](https://windowsserver.uservoice.com/forums/301869-powershell)
```

Since you're already editing this, can you add a colon to the end of line 12?

Reply:
As requested, so delivered.